environment/src/main/java/com/sequenceiq/environment/tags/v1/controller/AccountTagController.java

```diff
@@ -71,6 +71,12 @@ public AccountTagController(AccountTagService accountTagService,
     @CheckPermissionByAccount(action = AuthorizationResourceAction.ENVIRONMENT_READ)
     public AccountTagResponses list() {
         String accountId = ThreadBasedUserCrnProvider.getAccountId();
+        return listInAccount(accountId);
+    }
+
+    @Override
+    @CheckPermissionByAccount(action = AuthorizationResourceAction.ENVIRONMENT_READ)
```

Comment:
How is this limited to internalOnly?

Reply:
introduced an annotation for that, let's discuss if there is a simpler way to do this